pom.xml

```diff
@@ -197,13 +83,13 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
     <build>
         <sourceDirectory>src</sourceDirectory>
         <plugins>
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>build-helper-maven-plugin</artifactId>
-                <version>1.12</version>
```

Comment:
[meta] What is our policy wrt denoting versions for build plugins?
Advantage with providing an explicit version: We do not get injected newer versions automatically. The build is more stable.
Disadvantage with providing an explicit version: We do not get injected newer versions automatically. The build plugins are getting older and older without getting fixes

Reply:
I would go for not override versions, but I think that a general decision was not taken.

Reply:
ok, done